source/Tool/ToolWorld.hpp

```diff
@@ -0,0 +1,46 @@
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Copyright (c) 2023 Klemens D. Morgenstern
+//
+// Official repository: https://github.com/cppalliance/mrdox
+//
+
+#ifndef MRDOX_TOOL_TOOLWORLD_HPP
+#define MRDOX_TOOL_TOOLWORLD_HPP
+
+#include <memory>
+#include <vector>
+
+namespace clang {
+namespace mrdox {
+
+
+/** Holds the tool's global variables.
+    A single object of this type is used to manage
+    objects whose lifetime and order of destruction
+    is sensitive.
```

Comment:
Why did you remove the blank line I inserted after the brief? This is just wrong. The javadoc has a brief and no description:

![image](https://github.com/cppalliance/mrdox/assets/1503976/670868e4-9ae5-4966-85f3-576914bc5061)